modules/server/src/test/scala/higherkindness/mu/rpc/protocol/RPCMethodNameTests.scala

```diff
@@ -28,6 +28,11 @@ class RPCMethodNameTests extends RpcBaseTestSuite with BeforeAndAfterAll with Ch
 
   object RPCService {
 
+    import cats.kernel.instances.string._
```

Comment:
Should these imports be added by the annotation within the generated class? Not asking it for this PR, but in case we need a follow-up.

Reply:
Yep, I think I should